test/uk/gov/hmrc/merchandiseinbaggagefrontend/pagespecs/GoodsRouteDestinationPageSpec.scala

```diff
@@ -30,13 +30,13 @@ class GoodsRouteDestinationPageSpec extends DeclarationDataCapturePageSpec[YesNo
     behave like aPageWhichRenders(path, givenAnExportJourneyIsStarted(), exportTitle)
     behave like aPageWhichRequiresADeclarationJourney(path)
 
-    behave like aDataCapturePageWithConditionalRoutingWithoutPersistence(
+    behave like aDataCapturePageWithConditionalRouting(
       path, givenAnImportJourneyIsStarted(), Yes, CannotUseServiceIrelandPage.path)
-    behave like aDataCapturePageWithConditionalRoutingWithoutPersistence(
+    behave like aDataCapturePageWithConditionalRouting(
       path, givenAnImportJourneyIsStarted(), No, ExciseAndRestrictedGoodsPage.path, "and declarationType is Import")
-    behave like aDataCapturePageWithConditionalRoutingWithoutPersistence(
```

Comment:
I don't think this test should be removed, as discussed above changing the routing from the page wasn't part of the A/C